### PR TITLE
Update install-app-guard.md

### PR DESCRIPTION
**Pull request by manoth-msft**
Updated and clarified the references to the Microsoft 365 Apps plus various channels (removing the generic 'Office' term which leads to confusion if e.g. Office 2019 is supported).

### CODE REVIEW

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ Microsoft Defender Application Guard for Office (Application Guard for Office) h
 ### Minimum software requirements
 
 * **Windows**: Windows 10 Enterprise edition, Client Build version 2004 (20H1) build 19041 or later. All versions of Windows 11 are supported.
-* **Office**: Office Current Channel and Monthly Enterprise Channel, Build version 2011 16.0.13530.10000 or later. Office Semi-Annual Enterprise Channel, Build version 2108 or later. Both 32-bit and 64-bit versions of Office are supported.
+* **Office**: Microsoft 365 apps with build 16.0.13530.10000 or later. For Current Channel and Monthly Enterprise Channel installations, this equals to version 2011. For Semi-Annual Enterprise Channel and Semi-Annual Enterprise Channel (Preview), minimum version is 2108 or later. Both 32-bit and 64-bit versions are supported.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
* **Office**: Microsoft 365 Apps with build 16.0.13530.10000 or later. For Current Channel and Monthly Enterprise Channel installations, this equals to version 2011. For Semi-Annual Enterprise Channel and Semi-Annual Enterprise Channel (Preview), the minimum version is 2108 or later. Both 32-bit and 64-bit versions are supported.
```

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 2]
```diff
@@ -35,11 +35,11 @@ Microsoft Defender Application Guard for Office (Application Guard for Office) h
...
 * **Update package**: Windows 10 cumulative monthly security update [KB4571756](https://support.microsoft.com/help/4571756/windows-10-update-KB4571756)
 
 For detailed system requirements, refer to [System requirements for Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard). Also, please refer to your computer manufacturer's guides on how to enable virtualization technology.
-To learn more about Office update channels, see [Overview of update channels for Microsoft 365](/deployoffice/overview-update-channels).
+To learn more about Microsoft 365 apps update channels, see [Overview of update channels for Microsoft 365 Apps](/deployoffice/overview-update-channels).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
To learn more about Microsoft 365 Apps update channels, see [Overview of update channels for Microsoft 365 Apps](/deployoffice/overview-update-channels).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @manoth-msft, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@schmurky Gentle reminder. Thanks

**Pull request merged**